UEBPrep/app/controllers/homepage_controller.rb

```diff
@@ -1,4 +1,25 @@
 class HomepageController < ApplicationController
-  def index
```

Comment:
Index, create, update, (along with delete) are conventional methods for a function that have conventional definitions and uses, if I call the create function on the {insert-name-here}controller, I expect a new {insert-name-here}. For that reason, and to assist in code readability and understandably, we don't want to overload or replace those functions with different behavior unless we have very good reason to.